### DISCUSSION TITLE
list of .mp4 files with ffmpeg via rtmp

### DISCUSSION THREAD

**Discussion by 918483** [savonet/liquidsoap#2454] [User Support]
Hello everyone, I would like to know how do I transmit a list of .mp4 files with ffmpeg to rtmp via liquidsoap ?
Could someone help showing me some script?

**Comment by toots** [comment]
Hi,

You want to receive a list of mp4 files in a liquidsoap instance and play them as rtmp?

I suppose using the [http server](https://www.liquidsoap.info/doc-2.0.4/server.html) you could implement a http endpoint to receive them. Then you could push them to a [request.queue](https://www.liquidsoap.info/doc-2.0.4/reference.html#request.queue)

**Reply by 918483** [reply]
I just need a way to stream the list of .mp4 files with liquidsoap to rtmp or rtmps

**Reply by 918483** [reply]
I already have the list of files created on the server and the mp4 files I wanted to use liqudisoap for it to run the playlist by fading texts and these things
